agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3507,8 +3508,8 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
         }
 
         std::array<beerocks::message::sWifiChannel, beerocks::message::SUPPORTED_CHANNELS_LENGTH>
-            supported_channels_arr;
-        std::copy_n(supported_channels.begin(), supported_channels_arr.size(),
+            supported_channels_arr{{}};
+        std::copy_n(supported_channels.begin(), supported_channels.size(),
```

Comment:
This is an actual fix, so the "preparative commit" part of the commit message is not really needed. (nitpick, obviously)